src/components/Integrations/BigQuery/FreeAccount/FreeAccount.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { Fragment, useState } from 'react';
+import './styles/index.css';
+import { useIntl } from 'react-intl';
+import SafeRedirect from '../../../SafeRedirect';
+
+interface IFreeAccountProps {}
+
+const FreeAccount: React.SFC<IFreeAccountProps> = (props) => {
```

Comment:
Please note that React.SFC is deprecated from version 16.7 onwards, we are using react 17.0.5 right now https://github.com/FromDoppler/doppler-webapp/blob/master/package.json#L15
```suggestion
const FreeAccount: React.FunctionComponent<IFreeAccountProps> = (props) => {
```

Reply:
For components we don't usually use TypeScript. We just only use JavaScript. 
I think that in this case just:
```
const FreeAccount = () => {
..........
```
it's ok.

Reply:
Yes but he likes typescript, I think is ok too.